js_modules/dagster-ui/packages/ui-core/src/runs/useRunsFeedEntries.tsx

```diff
@@ -11,9 +11,9 @@ import {gql, useQuery} from '../apollo-client';
 import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
 import {RunsFilter} from '../graphql/types';
 
-const PAGE_SIZE = 25;
+const PAGE_SIZE = 29;
```

Comment:
Why 29?

Reply:
Ahh this is because of a bug on the graphql side that causes it to return 1 more row than requested. Let me see if i can find the root cause and fix it there so this is less janky